aws/logs_monitoring/lambda_function.py

```diff
@@ -100,11 +100,24 @@ def get_bool_env_var(envvar, default):
 DD_FORWARD_LOG = get_bool_env_var("DD_FORWARD_LOG", "true")
 
 ## @param DD_USE_TCP - boolean - optional -default: false
-## Change this value to `true` to send your logs and metrics using the HTTP network client
+## Change this value to `true` to send your logs and metrics using the TCP network client
 ## By default, it uses the HTTP client.
 #
 DD_USE_TCP = get_bool_env_var("DD_USE_TCP", "false")
 
+## @param DD_USE_COMPRESSION - boolean - optional -default: true
```

Comment:
Worth mentioning that's only valid for HTTP

Reply:
Added here: https://github.com/DataDog/datadog-serverless-functions/pull/190/files#diff-618731ddf1c446ef45d09613cb4b189bR109